### DISCUSSION TITLE
Fast Stable Diffusion with everything included (google colab notebook)

### DISCUSSION THREAD

**Discussion by Bullseye-StableDiffusion** [TheLastBen/fast-stable-diffusion#1532] [Show and tell]
Hello!
I added some things to the base fast stable diffusion notebook.
It can be found for download here: https://github.com/Bullseye-StableDiffusion/stable_diffusion_webui_allinone_dreambooth/blob/main/fast_stable_diffusion_with_everything_included.ipynb .
You can even train with the Dreambooth extension.
Let me to know if you wish to see other features.

**Comment by gilroff** [comment]
what did you add?

**Reply by Bullseye-StableDiffusion** [reply]
LoRA, hypernetwork, textual inversion, a good model (dreamshaper) and some extensions.

**Reply by gilroff** [reply]
cool!

**Comment by SDPDVADER** [comment]
Good idea, where can we access the google collab notebook?

**Reply by Bullseye-StableDiffusion** [reply]
On the provided link. Right click on "Raw" -> "Save link as..." and this is the notebook file. Now you can upload and use it on Google Colab.

**Comment by modulator** [comment]
Yeah this is great!  But how do I turn this code into a working colab notebook?

**Reply by Bullseye-StableDiffusion** [reply]
On the provided link. Right click on "Raw" -> "Save link as..." and this is the notebook file. Now you can upload and use it on Google Colab.

**Reply by SDPDVADER** [reply]
Better yet this link 

https://colab.research.google.com/github/Bullseye-StableDiffusion/fast_stable_diffusion_with_everything_included/blob/main/fast_stable_diffusion_with_everything_included.ipynb

**Reply by Bullseye-StableDiffusion** [reply]
> Better yet this link
> 
> https://colab.research.google.com/github/Bullseye-StableDiffusion/fast_stable_diffusion_with_everything_included/blob/main/fast_stable_diffusion_with_everything_included.ipynb

Yea, or like this. :))

**Comment by SDPDVADER** [comment]
![image](https://user-images.githubusercontent.com/123031699/220325966-a2ac0efa-405d-4c20-9eeb-7fe7f462c94d.png)
something is happening

**Reply by Bullseye-StableDiffusion** [reply]
Nah, it works fine. I tested it now. Click on this link https://github.com/Bullseye-StableDiffusion/stable_diffusion_webui_allinone_dreambooth/blob/main/fast_stable_diffusion_with_everything_included.ipynb, then right click on "Raw" -> "Save link as..." and then upload it to google colab.
